app/models/course.rb

```diff
@@ -457,4 +462,14 @@ def validate_qualification
   def set_applications_open_from
     self.applications_open_from ||= recruitment_cycle.application_start_date
   end
+
+  def validate_applications_open_from
+    if provider.present?
```

Comment:
dont think course can live with a provider

Reply:
Same issue that we tried to solve yesterday. It's an issue with the belongs_to matcher which makes the course belongs to provider test fail without this line of code.